src/posts/datatree/index.md

```diff
@@ -0,0 +1,104 @@
+---
+title: 'Xarray x NASA: xarray.DataTree for hierarchical data structures'
+date: '2024-12-04'
+authors:
+  - name: Tom Nicholas
+    github: tomnicholas
+  - name: Owen Littlejohns
+    github: owenlittlejohns
+  - name: Matt Savoie
+    github: flamingbear
+  - name: Eni Awowale
+    github: eni-awowale
+  - name: Alfonso Ladino
+    github: aladinor
+  - name: Justus Magin
+    github: keewis
+  - name: Stephan Hoyer
+    github: shoyer
+summary: 'The new xarray.DataTree class allows working with netCDF/Zarr groups, brought to you in collaboration with NASA!'
+---
+
+## tl;dr
+
+[`xarray.DataTree`](https://docs.xarray.dev/en/stable/user-guide/data-structures.html#datatree) has been [released](https://github.com/pydata/xarray/discussions/9680) in [v2024.10.0](https://github.com/pydata/xarray/releases/tag/v2024.10.0), and the prototype [`xarray-contrib/datatree` repository](https://github.com/xarray-contrib/datatree) archived, after collaboration between the xarray team and the [NASA ESDIS project](https://www.earthdata.nasa.gov/about/esdis). 🤝
+
+## Why trees?
+
+Xarray users have been [asking](https://github.com/pydata/xarray/issues/4118) for a way to handle multiple netCDF4 groups [since at least 2016](https://github.com/pydata/xarray/issues/1092). Such netCDF4/Zarr groups are the on-disk representation of a general problem of handling hierarchies of related but non-alignable array data. Real-world datasets (such as [climate model intercomparisons](https://medium.com/pangeo/easy-ipcc-part-1-multi-model-datatree-469b87cf9114)) often fall into this category, and users wanted a way to work with such hierarchical data in-memory and a way to interact with it on disk.
```

Comment:
It would be nice to describe the motivation for trees in a way that makes sense to readers who don't know what a netCDF group is.

I usually use the analogy of a directory struture. DataTree allows for organizing heterogeneous collections of scientific data in the same way that a nested directory structure facilitates organizing large numbers of files on disk.

I would also consider mentioning at least three distinct use-cases that illustrate what DataTree will enable. Perhaps:
- climate model intercomparisons
- image pyramids
- complex satellite data or geospatial rasters?

Feel free to adapt my words here (or I can also take a pass at this later).

Reply:
Good point, I'll add those suggestions.

Reply:
If you end up enumerating here, it can't hurt to reiterate; simple, convenient access to an entire hierarchical file.

Reply:
Added in https://github.com/xarray-contrib/xarray.dev/pull/736/commits/22264dc4917a9e6e1aaffd50e67cc081087f138c